appinfo/info.xml

```diff
@@ -15,7 +15,7 @@
 		<prelogin/>
 	</types>
 	<category>security</category>
-	<screenshot></screenshot>
+	<screenshot>https://raw.githubusercontent.com/owncloud/brute_force_protection/7258b38283af4e3fd8a0f03084694b6ff6339505/screenshots/brute-force-protection.jpg</screenshot>

```

Comment:
Q: I looked at the picture and asked me following questions:
- Is the picture a placeholder or a (final) defined one
- If defined,
has it passed owncloud marketing approval or at least pending 
is it ok that it contains watermarks 
- Very personal opinion - it looks ugly 
So I hope it is a paceholder

Reply:
@mmattel It would be really good if someone suggests any screenshot. It is not approved by the marketing team yet. We can change it.

Reply:
@pmaier @hurradieweltgehtunter